Fix: refactor NotFound page to use Chakra UI components

LinHuiqing opened the pull request:
## Problem

Closes #8

## Solution

**Improvements**:

- Refactor code on NotFound page to use Chakra UI.

## Before & After Screenshots

**BEFORE**:
<img width="1792" alt="Screenshot 2021-09-07 at 9 48 37 AM" src="https://user-images.githubusercontent.com/37061143/132277087-860de342-5325-47e5-96c5-98ac850e69c8.png">

**AFTER**:
<img width="1792" alt="Screenshot 2021-09-07 at 10 30 03 AM" src="https://user-images.githubusercontent.com/37061143/132277122-f527c3ac-9dac-4bac-8a1f-7dcabc13dcbb.png">

Commit:
issue 8: convert NotFound page to Chakra, rm box__ghost

LinHuiqing committed:
fix: minor edits to NotFound copy
remove "Whoops!" line
increase spacing between heading, text and button
remove smiley in copy

LoneRifle reviewed (dismissed):
The commit log is extremely messy, making this very difficult to review. Figure a way to make a new branch that contains just the changes that you did, and push that new branch to `refactor/to-chakra`

LoneRifle commented on client/src/pages/NotFound/NotFound.styles.scss:
```diff
@@ -12,286 +12,3 @@ html, body {
 * {
   box-sizing: border-box;
 }
```
consider either removing or moving this into the component itself. This should allow us to the scss file

LinHuiqing committed:
Cleanup NotFound styling

remove NotFound scss styling file
change NotFound page extension from .jsx to .tsx

LinHuiqing committed:
Fix error caught by ci Lint

Address following lint error:
Missing return type on function @typescript-eslint/explicit-module-boundary-types

LoneRifle reviewed (approved):
lgtm
